server/__tests__/index_test.js

```diff
@@ -0,0 +1,27 @@
+const request = require('supertest');
+const app = require('../index')
+
```

Comment:
One more general comment not closely related to the line it's on :D 

Is there a reason some of the .js files have 4 spaces indentation and some have 2?

Reply:
Good question. I think it may be something I need to change in the code editor. I'll look into it